staff/edit-courses/edit-courses.php

```diff
@@ -0,0 +1,143 @@
+<!DOCTYPE html>
+<?php
+	require_once "../../common.php";
+	require_once "course-common.php";
+	require_staff();
+?>
+<html lang="en">
+<head>
+    <title>Staff - CourseCorrect</title>
+    <meta charset="utf-8">
+	<link rel="icon" href="../favicon.ico">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="../../libs/bootstrap.min.css">
+	<script src="../../libs/jquery.slim.min.js"></script>
+	<script src="../../libs/popper.min.js"></script>
+	<script src="../../libs/bootstrap.min.js"></script>
+	<script src="../../libs/autoComplete.min.js"></script>
+	<link rel="stylesheet" href="../../libs/autoComplete.02.css">
+	<link rel="stylesheet" href="../../libs/fontawesome.min.css">
+	<script src="edit-course.js"></script>
+	<style>
+		.form-control::placeholder {
```

Comment:
Is this working for you? I was trying something similar on my page and couldn't get it to work.